docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
 # UCI API
   uci-api-service:
     build: 
-      context: ./uci-apis/v2/uci
+      context: ./uci-apis/
```

Comment:
Now that we have proper versioning being used in UCI services, we should pull it from the docker hub rather than build our own image. @chinmoy12c can you confirm whether every service is now being pushed to the docker hub or not?

Reply:
Resolving this comment for now - Let's use this for now and make a new issue to only use docker hub images as a community program issue.